src/Form/SettingsForm.php

```diff
@@ -74,6 +77,18 @@ public function buildForm(array $form, FormStateInterface $form_state): array {
       '#process_default_value' => FALSE,
     ];
 
+    $form['introduction_text'] = [
+      '#type' => 'text_format',
+      '#title' => $this->t('Introduction text'),
+      '#default_value' => $introduction_text['value'] ?? '',
+      '#allowed_formats' => !empty($formats) ? array_keys($formats) : [],
```

Comment:
This is not how it should be done. The filter module has a dedicated setting that will show or not the fallback format (always_show_fallback_choice). We don't need to pass any allowed formats at all.

Reply:
Ok